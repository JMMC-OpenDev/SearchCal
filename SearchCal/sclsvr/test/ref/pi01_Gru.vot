version https://git-lfs.github.com/spec/v1
oid sha256:1d96bd4054d5b885d50264e2de9a766fe176badd1e630287eccf0fc2dedebcd8
size 71786
