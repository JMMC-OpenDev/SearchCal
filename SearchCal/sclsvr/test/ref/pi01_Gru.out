version https://git-lfs.github.com/spec/v1
oid sha256:38be84eac6b6f5c46731dfd34b24f38bcbfba14473b9222f161f020300d12a61
size 206809
